app/src/main/java/org/nv95/openmanga/providers/staff/Languages.java

```diff
@@ -14,9 +14,12 @@
     public static final int MULTI = 4;
     public static final int VIE = 5;
     public static final int FR = 6;
+    public static final int ES = 7;
 
     public static int fromLocale(Locale locale) {
         switch (locale.getLanguage()) {
+            case "es":
+                return ES;
```

Comment:
the app has no translation for es, please add translate in values-es or delete this line

Reply:
> the app has no translation for es, please add translate in values-es or delete this line

This is ok